test/bench/node.js

```diff
@@ -1,3 +1,3 @@
-require('reify');
+require('@babel/register');
```

Comment:
Noooo not babel register..., please let me debug untranspiled code.... ;)

Just bump to node 14 to ecma script 2020 support.

Reply:
Node.js does not support our style of module imports (a mix of es modules and commonjs).

`reify` was meant to add that support, however it crashes on ECMA 2020 syntax. The module has also not been updated for 2 years.

I have added an explicit option to `@babel/register` to only use transforms that are needed by Node 14.

Reply:
Got it, that is not too bad. FWIW, there is `esm` too (reify alternative) it may be more up to date.